### PR TITLE
Add version support for Java and Python in CloudSlang

### LINKED ISSUE #630: Support backward compatibility - versioning
- backward compatibility *may* be affected by the cloudslang versioning plans (e.g. adding version tag in DSL)
- visual data in flows  - [SOLVED](https://github.com/CloudSlang/cloud-slang/issues/643)

### PR DESCRIPTION
**Pull request by genadi-hp**
fixes #630
Sign-off: Genadi Rabinovich, @genadi-hp, genadi@hpe.com

### CODE REVIEW

File: cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/bindings/scripts/ScriptExecutor.java [hunk 1]
```diff
@@ -11,42 +11,29 @@
...
-        }
+    public Map<String, Serializable> executeScript (String script, Map<String, Serializable> callArguments) {
+        return pythonRuntimeService.exec(Collections.<String>emptySet(), script, callArguments);
     }
 
-    private void prepareInterpreterContext(Map<String, Serializable> context) {
-        for (Map.Entry<String, Serializable> entry : context.entrySet()) {
-            interpreter.set(entry.getKey(), entry.getValue());
-        }
+    public Map<String, Serializable> executeScript (Set dependencies, String script, Map<String, Serializable> callArguments) {
```
Comment (Bonczidai) [inline cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/bindings/scripts/ScriptExecutor.java]: this should be `Set<String> dependencies` right?

File: cloudslang-runtime/src/test/java/io/cloudslang/lang/runtime/bindings/scripts/ScriptEvaluatorTest.java [hunk 2]
```diff
@@ -39,23 +51,24 @@
...
     public ExpectedException exception = ExpectedException.none();
 
-    @InjectMocks
-    private ScriptEvaluator scriptEvaluator = new ScriptEvaluator();
+    @Autowired
+    private ScriptEvaluator scriptEvaluator;
 
-    @Mock
-    private PythonInterpreter pythonInterpreter;
+    private static PythonInterpreter evalInterpreter = mock(PythonInterpreter.class);
```
Comment (Bonczidai) [inline cloudslang-runtime/src/test/java/io/cloudslang/lang/runtime/bindings/scripts/ScriptEvaluatorTest.java]: isn't error prone making this field static?

File: cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/bindings/scripts/ScriptExecutor.java (hunk 1 above)
Comment (genadi-hp) [inline cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/bindings/scripts/ScriptExecutor.java]: Fixed

File: cloudslang-runtime/src/test/java/io/cloudslang/lang/runtime/steps/ActionStepsTest.java [hunk 3]
```diff
@@ -78,7 +86,7 @@ public void setUp() {
         nonSerializableExecutionData = new HashMap<>();
     }
 
-    @Test(timeout = DEFAULT_TIMEOUT)
+    @Test//(timeout = DEFAULT_TIMEOUT)
```
Comment (Bonczidai) [inline cloudslang-runtime/src/test/java/io/cloudslang/lang/runtime/steps/ActionStepsTest.java]: IMHO you can remove this timeout completely

**Comment by Bonczidai** [comment]
- please replace `hpe` words in the PR with something else like `cloudslang`
- is the `test-mvn-repo/hpe` folder under system tests resources committed by mistake?

**Comment by genadi-hp** [comment]
test-mvn-repo/hpe is a part of test  maven local repository we use for maven tests. i will rename this hpe to cloudslang

**Comment by genadi-hp** [comment]
Hi, I moved all hpe to cloudslang

**Pull request closed without merging**